git-hg/git-hg.sh

```diff
@@ -16,41 +16,41 @@
 git clone https://github.com/frej/fast-export.git
```

Comment:
Agreed that `-eu` is a nice default, it's better to explicitly say `|| true` if you want to ignore an error.